api/code/prerenderingService/flush/src/index.ts

```diff
@@ -12,7 +12,7 @@ const documentClient = new DocumentClient({
 export const handler = createHandler(
     flushPlugins({
         storageOperations: createPrerenderingServiceStorageOperations({
-            table: table => {
+            table: (table: any) => {
```

Comment:
It's actually `TableModifier`, not `any`. But on the other hand, not sure why this is needed in the first place because I can see the types defined in `packages/api-prerendering-service-so-ddb/src/types.ts`.

Reply:
The same question goes for the same issue in other files.

Reply:
I will check that, maybe `createPrerenderingServiceStorageOperations` has not that well-written declaration and type inferring fails.

Reply:
Problem was in fact with env variable which may possibly be undefined, and TS couldn't properly inference type.
![image](https://user-images.githubusercontent.com/2473784/147458786-06bf8df8-36c7-4906-a5aa-2582e5bcbd5f.png)